packages/scss/src/theming/_breakpoints.scss

```diff
@@ -1,10 +1,11 @@
 $breakpoints: (
-	xs:  576px,
-	sm:  768px,
-	md:  992px,
+	xs: 576px,
+	sm: 768px,
+	md: 992px,
 	lg: 1200px,
 	xl: 1350px,
 	xxl: 1500px,
 	xxxl: 1650px,

```

Comment:
Are these arbitrary? If they are device sizes, shouldn't they be called as such ?